sqlmesh/dbt/model.py

```diff
@@ -172,7 +172,7 @@ def model_kind(self, target: TargetConfig) -> ModelKind:
                     logger.warning(
                         "SQLMesh incremental by time strategy is not compatible with '%s' incremental strategy in model '%s'. Supported strategies include %s.",
                         strategy,
-                        self.sql_name,
+                        self.full_sql_name,
```

Comment:
Why full name in all error messages? I thought the canonical name was `sql_name`

Reply:
`sql_name` requires the context object. For loading, I thought using the fqn could be a substitute. The other option I considered was package.model (e.g. sushi.items). What do you think?